src/node_process.h

```diff
@@ -4,6 +4,7 @@
 #if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
 
 #include "v8.h"
+#include "debug_utils-inl.h"
```

Comment:
We don’t include `-inl.h` headers from `.h` headers, that’s missing the point of having them separate. You probably want to move the `ProcessEmitWarning` definition into a new `node_process-inl.h` headers.

Reply:
@addaleax Done, PTAL.